quality_of_service_demo/README.md

```diff
@@ -156,8 +156,8 @@ and the "Deadline missed" messages will no longer be printed.
 This demo shows how to get a notification when a subscription loses a message.
 
 This feature is not available in all RMW implementations.
-`rmw_cyclonedds_cpp` and  `rmw_connextdds` do support this feature.
-CycloneDDS partially implements the feature and it only triggers the event under some limited circumstances, thus it's recommended to try the demo with Connext.
+`rmw_zenoh_cpp`, `rmw_cyclonedds_cpp` and  `rmw_connextdds` do support this feature.
```

Comment:
i think `RMW_EVENT_MESSAGE_LOST` is supported in rmw_fastrtps, i am not sure why that is not listed here?

https://github.com/ros2/rmw_fastrtps/blob/21e6a44a2ef25ce51ece019b255427a0248fad65/rmw_fastrtps_shared_cpp/src/custom_subscriber_info.cpp#L449


```suggestion
`rmw_fastrtps_cpp`, `rmw_zenoh_cpp`, `rmw_cyclonedds_cpp` and  `rmw_connextdds` do support this feature.
```

Reply:
Agreed.  I also think it is premature to add in `rmw_zenoh_cpp` here, as it is not a Tier-1 RMW.